src/openrct2-ui/interface/Window.cpp

```diff
@@ -478,8 +478,8 @@ static bool WindowOtherWheelInput(rct_window& w, WidgetIndex widgetIndex, int32_
         case WindowWidgetType::Spinner:
             buttonWidgetIndex = wheel < 0 ? widgetIndex + 1 : widgetIndex + 2;
             expectedType = WindowWidgetType::Button;
-            expectedContent[0] = ImageId(STR_NUMERIC_UP, FilterPaletteID::PaletteNull);
-            expectedContent[1] = ImageId(STR_NUMERIC_DOWN, FilterPaletteID::PaletteNull);
+            expectedContent[0] = ImageId(STR_NUMERIC_UP);
```

Comment:
What is this even doing? Why are we turning a string into an image.

Reply:
It checks if the next button has a + image, so that it knows it’s dealing with a spinner/stepper.

Reply:
Urgh this is messy code. Its actually checking that the next button has a + string so that it knows its dealing with a spinner stepper. We should really split this code in two one for the image button and one for the string button then we aren't trying to stuff a string in the ImageId and compare a ImageId with a string.

Reply:
I have pushed a refactor.